src/Adapters/EventSource.Netstandard13/EventSourceTelemetryModule.cs

```diff
@@ -218,5 +229,65 @@ private void EnableAsNecessary(EventSource eventSource)
                 }
             }
         }
+
+        /// <summary>
+        /// Returns true when eventSource satisfied the rule; false otherwise. Returns false when either is null.
+        /// </summary>
+        /// <param name="eventSource">The target event source.</param>
+        /// <param name="rule">The naming rule to be used for matching.</param>
+        private bool IsEventSourceNameMatch(EventSource eventSource, EventSourceNamingMatchRuleBase rule)
+        {
+            if (rule == null || eventSource == null)
+            {
+                return false;
+            }
+
+            if (!rule.PrefixMatch)
+            {
+                return string.Equals(eventSource.Name, rule.Name, StringComparison.Ordinal);
+            }
+            else
+            {
+                return IsEventSourceNameMatch(eventSource?.Name, rule?.Name);
+            }
+        }
+
+        /// <summary>
+        /// Returns true when eventSourceName satisfied the rule; false otherwise. Returns false when either is null.
+        /// </summary>
+        /// <param name="eventSourceName"></param>
+        /// <param name="rule"></param>
+        /// <returns></returns>
+        internal bool IsEventSourceNameMatch(string eventSourceName, string rule)
+        {
+            if (string.IsNullOrEmpty(eventSourceName) || string.IsNullOrEmpty(rule))
```

Comment:
you've added two conditions to check that was just checked before. Why not simply call `eventSourceName.StartsWith(rule)` instead of defining whole new method?

Reply:
Yup. That is a good point. It is simple enough to call starts with directly now.